package.json

```diff
@@ -71,6 +71,7 @@
     "otplib": "7.0.0",
     "po2json": "0.4.5",
     "poolee": "1.0.1",
+    "punycode": "^2.1.0",
```

Comment:
remove the `^` before the `2`

Reply:
do we need to change  this as well , if yes what all we need to do ?
https://github.com/mozilla/fxa-auth-server/blob/86de08ba33540a1720cc9974b03e189394fa9105/npm-shrinkwrap.json#L81

"punycode": {
                  "version": "1.4.1",
                  "from": "https://registry.npmjs.org/punycode/-/punycode-1.4.1.tgz",
                  "resolved": "https://registry.npmjs.org/punycode/-/punycode-1.4.1.tgz"
                }

Reply:
would have to run `npm run shrink` for that to regenerate that file